packages/devtools_app/lib/src/screens/performance/panes/rebuild_stats/rebuild_stats.dart

```diff
@@ -206,9 +210,16 @@ class _RebuildTableState extends State<RebuildTable> {
             ValueKey<String?>('${location.location.id}'),
         defaultSortColumn: _metricsColumns.first,
         defaultSortDirection: sortDirection,
-        onItemSelected: (item) {
-          // TODO(jacobr): navigate to selected widget in IDE or display the
-          // content side by side with the rebuild table.
+        onItemSelected: (item) async {
+          final location = item?.location;
+          if (location?.path != null) {
+            await _service?.postEvent('ToolEvent', 'navigate', <String, Object>{
```

Comment:
Can you add this as an extension method in this file? https://github.com/flutter/devtools/blob/master/packages/devtools_app_shared/lib/src/service/service_utils.dart/#L7

We have had DevTools extension authors inquiring about how to do this, so would be good to expose this there.

Reply:
Moved to `devtools_app_shared` - let me know if that interface looks good.

How does versioning work here? devtools_app seems to pick up the changes to the shared package, but I see that the dependency in devtools_app is for `devtools_app_shared: ^0.2.0-dev.0` - does this need to be updated?